### ISSUE TITLE
Bluetooth & Switch controller suport ? 

### ISSUE THREAD

**Issue by lewilou22** [pdaxrom/usb2n64-adapter#1]
Bluetooth & Switch controller support ? 

just a thought

**Comment by pdaxrom** [comment]
Do you mean gamepads with Bluetooth? It should be possible if enable support for a bluetooth adapter (in tinyusb) or use Raspberi Pi Pico W

**Comment by lewilou22** [comment]
Yes exactly!   I Have a Pico W.  Mostly new with this. Trying to look into what librarys to use with the Pi Pico W to get bluetooth connection working.

**Comment by GK6475** [comment]
Could it be updated to support the Switch Online N64 Controller via wired connection? I've been noticing that the BlueRetro adapters have a lot of latency.
